portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormFields.path

```diff
@@ -63,6 +63,11 @@
 	<td>//p[contains(text(),'${key_fieldLabel}')] | //label[contains(text(),'${key_fieldLabel}')] | //legend[contains(text(),'${key_fieldLabel}')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>LABEL_BY_ROW</td>
```

Comment:
@lima-rafa, please rename to `FIELD_LABEL_BY_ROW`

Reply:
OK